installer/charts/tssc-infrastructure/templates/postgres/pgsql-service.yaml

```
@@ -0,0 +1,115 @@
+    app: {{ $podName }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $podName }}
+  namespace: {{ $inst.namespace }}
+  annotations:
+    app.kubernetes.io/part-of: tssc
+spec:
+  selector:
+    matchLabels: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)